doc/releases/reef.rst

```diff
@@ -6,6 +6,428 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
 
+v18.2.3 Reef
+============
+
+This is the third backport release in the Reef series. We recommend that all users update to this release.
+
+Notable Changes
+---------------
+
+* RADOS: This release fixes a bug (https://tracker.ceph.com/issues/61948) where pre-reef clients were allowed
+  to connect to the `pg-upmap-primary` (https://docs.ceph.com/en/reef/rados/operations/read-balancer/)
+  interface despite users having set `require-min-compat-client=reef`, leading to an assert in the osds
+  and mons. You are susceptible to this bug in reef versions prior to 18.2.3 if 1) you are using an osdmap
+  generated via the offline osdmaptool with the `--read` option or 2) you have explicitly generated pg-upmap-primary
+  mappings with the CLI command. Please note that the fix is minimal and does not address corner cases such as
```

Comment:
@ljflores Are we certain that the online read balancer isn't affected?  I see the same in your email to ceph-users:
> We have discovered a bug with the pg-upmap-primary interface (related to the offline read balancer [1])

but isn't the balancer capable of sending `ceph osd pg-upmap-primary` commands to the monitor itself as part of executing the plan, without the user "explicitly generating pg-upmap-primary mappings with the CLI command?

Reply:
@idryomov the online read balancer is not in reef. It is only offered in squid.

Reply:
We will mention this in any squids-specific release note, although the fixes are being ported there before the GA version.